package.json

```diff
@@ -25,7 +25,7 @@
     "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
     "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
     "test": "jest",
-    "test-cli": "cd acceptance-tests && yarn test-headless --stream",
+    "test-cli": "yarn --cwd 'acceptance-tests' test-ci",
```

Comment:
The `--cwd` sets the `cwd` of the `yarn` command so we don't need to `cd`

Reply:
Should we make it so that we can also run the other variations of the acceptance tests from the root of the project? Like the QA and debug ones, for example

Reply:
Good call, I like that